java/code/webapp/WEB-INF/pages/systems/sdc/overview.jsp

```diff
@@ -8,6 +8,7 @@
 <body>
 <%@ include file="/WEB-INF/pages/common/fragments/systems/system-header.jspf" %>
 
+  <rhn:require acl="not system_is_proxy() or not system_has_foreign_entitlement()">
```

Comment:
Do we need to filter also proxy in here? Is not enough to filter by foreign entitlement?

Reply:
same here...